kubernetes/coredns.yaml.sed

```diff
@@ -170,3 +170,6 @@ spec:
   - name: dns-tcp
     port: 53
     protocol: TCP
+  - name: metrics
+    port: 9153
+    protocol: TCP
```

Comment:
nit: newline

Reply:
Thanks, fixed I think. Sorry for the extra git noise, got a bit stuck.